command/agent/consul/int_test.go

```diff
@@ -31,7 +31,6 @@ func testLogger() *log.Logger {
 // TestConsul_Integration asserts TaskRunner properly registers and deregisters
 // services and checks with Consul using an embedded Consul agent.
 func TestConsul_Integration(t *testing.T) {
-	t.Skip("-short set; skipping")
```

Comment:
Ah my bad I leaked this in. So I am not 100% sure it is this test but 85+% sure that it can leak the Consul agent sometimes. I noticed running make test many times that *1* consul agent sticks around and consumes a lot of CPU (70+%). May be worth looking into